tests/unit/specs/scripts/num.spec.js

```diff
@@ -80,6 +81,22 @@ describe(`number helper`, () => {
     expect(percent(0.2612)).toBe(`26.12%`)
   })
 
+  it(`should format a very big percent. Convert it to trillions`, () => {
+    expect(percentBig(`8719010560462937.9`)).toBe(`8,719.01 T %`)

```

Comment:
this is an internationalized number - is that going to be ok in the tests?

Reply:
For now it looks like yes